src/utils/sformat.fs

```diff
@@ -54,65 +54,73 @@ namespace Microsoft.FSharp.Text.StructuredFormat
 
     [<NoEquality; NoComparison>]
 #if COMPILER
+    type internal BoxRange = BoxRange of obj
```

Comment:
@saul  (I've done this kind of hack in the past but should have been scolded at the time!)